pages/receive/Receive.tsx

```diff
@@ -271,44 +271,46 @@ export function Receive() {
       )}
       {/* TODO: move to one place - this is all copied from LNURL-Pay */}
       {!invoice && enterCustomAmount && (
-        <TouchableWithoutFeedback
-          onPress={() => {
-            Keyboard.dismiss();
-          }}
+        <KeyboardAvoidingView
```

Comment:
Should we wrap this in a component ourselves to avoid spreading these behaviors across 10s of files maybe? They all look very similar...

Reply:
I agree, @im-adithya could you do this?

Reply:
Done!